core/src/apps/aptos/helper.py

```
@@ -3,10 +3,20 @@
 
 def aptos_address_from_pubkey(pub_key_bytes: bytes) -> str:
     # append single-signature scheme identifier
     payloads = pub_key_bytes + b"\x00"
     address = f"0x{hexlify(sha3_256(payloads).digest()).decode()}"
     return address
+
+
+def aptos_transaction_prefix(tx_type: int) -> bytes:
+    if tx_type == 0:
+        return TRANSACTION_PREFIX
+    elif tx_type == 1:
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)